pwa/src/templates/organizationDetail/OrganizationDetailTemplate.tsx

```diff
@@ -14,22 +13,28 @@ import * as React from "react";
 import * as styles from "./OrganizationDetailTemplate.module.css";
 import { GitHubLogo } from "../../assets/svgs/GitHub";
 import organizationLogo from "./../../assets/svgs/LogoRotterdam.svg";
-import { ExternalLinkIcon, CallIcon, EmailIcon } from "@gemeente-denhaag/icons";
 import { TableResultTemplate } from "../templateParts/resultsTemplates/table/TableResultTemplate";
-import { Table, TableBody, TableCell, TableHeader, TableRow } from "@gemeente-denhaag/table";
 import { TEMPORARY_COMPONENTS } from "../../data/components";
+import { ToolTip } from "../../components/toolTip/ToolTip";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEnvelope, faGlobe, faPhone } from "@fortawesome/free-solid-svg-icons";
+import { Tag } from "../../components/tag/Tag";
+import { useTranslation } from "react-i18next";
+import { GitLabLogo } from "../../assets/svgs/GitLab";
+import { navigate } from "gatsby";
 
 interface OrganizationDetailTemplateProps {
   organizationId: string;
 }
 
 export const OrganizationDetailTemplate: React.FC<OrganizationDetailTemplateProps> = ({ organizationId }) => {
   const [currentTab, setCurrentTab] = React.useState<number>(0);
+  const { t } = useTranslation();
 
   return (
     <Container layoutClassName={styles.container}>
       <div className={styles.headerContainer}>
-        <div className={styles.headerContent}>
+        <div className={styles.headerOrganizationDescription}>
```

Comment:
I'd make the gap between the organisation details and the data (incl. tags) bigger. Right now it's `--web-app-size-md`, maybe `-xl` would be better.